tools/retest/main.go

```diff
@@ -109,6 +114,7 @@ func commentsToCreate(statuses map[string]string, jobsToRetest []string, shouldR
 	for _, job := range jobsToRetest {
 		state := statuses[job]
 		if state == "pending" {
+			log.Printf("Not adding comment for job %q because it is already running", job)
```

Comment:
Let's mention what comment we have in mind, and not judge whether it's "already" or "still":
```suggestion
			log.Printf("Not issuing /test %q because the job is %s", job, state)
```